### PR TITLE
gRPC: master-to-grpc merge

### PR DESCRIPTION
**Pull request by ggezer**

### CODE REVIEW

File: underfs/hdfs/src/main/java/alluxio/UfsConstants.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
```
Comment (ggezer) [inline underfs/hdfs/src/main/java/alluxio/UfsConstants.java]: I think I overlooked this file since it just popped up on my staging area...

Comment (calvinjia) [reply underfs/hdfs/src/main/java/alluxio/UfsConstants.java]: I don't think this file is checked in?

Comment (bf8086) [reply underfs/hdfs/src/main/java/alluxio/UfsConstants.java]: This is from multi-hdfs feature. It should be generated from the template. Maybe we need to add it to .gitignore?

Comment (ggezer) [reply underfs/hdfs/src/main/java/alluxio/UfsConstants.java]: updated .gitignore

**Pull request merged**